src/controllers/orderController.js

```diff
@@ -0,0 +1,37 @@
+const orderService = require("../services/orderService");
+
+const checkOrder = async (req, res) => {
+  try {
+    const bookId = req.params.bookId;
+    const userId = req.user.id;
+
+    if (!userId || !bookId) {
+      throw new Error("Key Error");
+    }
```

Comment:
userId가 없는 경우는 이미 auth 모듈에서 에러 처리를 하고 있으니 제외해도 됩니다!
```suggestion
    if (!bookId) {
      throw new Error("Key Error");
    }
```